### DISCUSSION TITLE
ChatGPT weighs in

### DISCUSSION THREAD

**Discussion by dworkin** [dworkin/dgd#82] [General]
![chatgpt](https://user-images.githubusercontent.com/274867/219682713-aaad8237-bd3c-4616-87af-5645b8e042a8.png)

**Comment by Blain20** [comment]
Hi, Lars! :O)

**Comment by dworkin** [comment]
Indeed.  Perhaps it is more of a statement about how passé DGD is, than about the failure of embryonic AI.

**Comment by dworkin** [comment]
Here is one from DeepSeek V4.

<img width="833" height="557" alt="Image_1781191236517_75" src="https://github.com/user-attachments/assets/87bd9f7d-a1f2-4ebf-82b0-3884f3a4c4ab" />
